docs/sdk/api-reference.md

```diff
@@ -1097,10 +1006,10 @@ dataset = Dataset(dataset_id='dataset_id', user_id='user_id', app_id='app_id')
 all_dataset_inputs = list(dataset.list_inputs())
 ```
 
-### Dataset.\__iter\__()
+### Dataset.\__iter\__
 
 ```python
-def __iter__(self) -> Iterator[DatasetExportReader]
+Dataset.__iter__()
```

Comment:
@sanjaychelliah

Reply:
As this is internal method so i have removed this from doc